db/migrate/20220926164441_add_slug_to_article.rb

```diff
@@ -0,0 +1,21 @@
+class AddSlugToArticle < ActiveRecord::Migration[6.1]
+  def up
+    add_column :articles, :slug, :string, default: ''
```

Comment:
is the default: '' required ?

Reply:
When null is `false` we need to have some default value. This usually helps with the previous records which are missed with the slug.